website/pages/_app.js

```diff
@@ -0,0 +1,18 @@
+import { ThemeProvider } from '@strapi/design-system/ThemeProvider';
+import { lightTheme } from '@strapi/design-system/themes';
+import PropTypes from 'prop-types';
+
+function MyApp({ Component, pageProps }) {
+  return (
+    <ThemeProvider theme={lightTheme}>
+      <Component {...pageProps} />
+    </ThemeProvider>
```

Comment:
Can you add a skip to content component here please.

`import { SkipToContent } from '@strapi/design-system/Main';`

```suggestion
    <ThemeProvider theme={lightTheme}>
      <SkipToContent>Skip to content</SkipToContent>
      <Component {...pageProps} />
    </ThemeProvider>
```

Reply:
@HichamELBSI what does this component do ?